desktop-widgets/diveplanner.ui

```diff
@@ -163,73 +134,139 @@
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
-          <string>mbar</string>
+          <string>m</string>
          </property>
          <property name="minimum">
-          <number>689</number>
+          <number>-100</number>
          </property>
          <property name="maximum">
-          <number>1100</number>
+          <number>3000</number>
+         </property>
+         <property name="singleStep">
+          <number>10</number>
          </property>
         </widget>
        </item>
-       <item row="3" column="0">
-        <widget class="QSpinBox" name="atmHeight">
+       <item row="3" column="1">
+        <widget class="QSpinBox" name="ATMPressure">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
+         <property name="statusTip">
+          <string/>
+         </property>
          <property name="suffix">
-          <string>m</string>
+          <string>mbar</string>
          </property>
          <property name="minimum">
-          <number>-100</number>
+          <number>689</number>
          </property>
          <property name="maximum">
-          <number>3000</number>
-         </property>
-         <property name="singleStep">
-          <number>10</number>
+          <number>1100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
-        <widget class="QDoubleSpinBox" name="salinity">
+        <widget class="QComboBox" name="waterType">
+         <item>
+          <property name="text">
+           <string>Fresh water</string>
+          </property>
+         </item>
+         <item>
+          <property name="text">
+           <string>Sea water</string>
+          </property>
+         </item>
+         <item>
+          <property name="text">
+           <string>Custom</string>
+          </property>
+         </item>
+        </widget>
+       </item>
+       <item row="3" column="3">
+        <widget class="QDoubleSpinBox" name="customSalinity">
+         <property name="enabled">
+          <bool>false</bool>
+         </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
+         <property name="maximumSize">
+          <size>
+           <width>90</width>
+           <height>16777215</height>
+          </size>
+         </property>
+         <property name="toolTip">
+          <string extracomment="Custom water density"/>
+         </property>
          <property name="suffix">
-          <string>kg/ℓ</string>
+          <string>kg/l</string>
```

Comment:
I think this "ℓ" for liter is a Subsurface standard and we should continue to use it consistently.

Reply:
You missed this second "l"

Reply:
Damn. I'll fix it. Thank you!